summarization/hf_transformers/requirements.txt

```diff
@@ -0,0 +1,11 @@
+accelerate
+datasets >= 1.8.0
+sentencepiece != 0.1.92
+protobuf
+rouge-score
+nltk
+py7zr
+torch >= 1.3
+evaluate
+neptune-client
+transformers >= 4.22
```

Comment:
lets pin this to the actual version you've used

Reply:
On looking a bit, the problem is we have used the development version (4.22 which is yet to be released).

Our integration will be released with that version. I tried to see if there is an easy way to install the nightly version of `transformers` via pip but there isn't. You have to manually install in locally.